app/src/main/java/org/mozilla/focus/widget/DownloadListAdapter.java

```diff
@@ -113,7 +113,9 @@ public void onBindViewHolder(RecyclerView.ViewHolder viewHolder, int position) {
                 final int progress = (int) (100 * downloadInfo.getSizeSoFar() / downloadInfo.getSizeTotal());
                 holder.progressBar.setProgress(progress);
                 holder.progressBar.setVisibility(View.VISIBLE);
-                subtitle = progress + "%";
+                Double downloadedbytes = (downloadInfo.getSizeSoFar() / 1000000);
+                Double totalbytes = (downloadInfo.getSizeTotal() / 1000000);
+                subtitle = downloadedbytes.shortValue() + "Mb" + "/" + totalbytes.shortValue() + "Mb";
```

Comment:
Please use MB as in app/src/main/java/org/mozilla/focus/screenshot/ScreenshotViewerActivity.java